packages/react-lowcode/src/codegen/generation/generators/list/basic-table-generator.ts

```diff
@@ -54,10 +51,10 @@ export class BasicTableGenerator implements TableGenerator
                         if (tableHead && tableBody) {
                             const tableHeadRow = this.findElementByName(tableHead, tableDefinition.row.tagName.text);
                             const tableBodyRow = this.findElementByName(tableBody, tableDefinition.row.tagName.text);
-
+                            // to be able insert into iteration with right iterator
+                            const prefixIterator = (tableBodyRow?.parent as any)?.parameters[0]?.getText()
```

Comment:
I do not understand the intent of this line (what is the idea behind `prefixIterator`?. Please, explain.

Nullsafetines: try what happens `null[0]` ...

Reply:
For ex. In basic table `<TableBody>` consists of many `<TableRows>`.  So in `<TableBody>` you iterate through array and create `<TableRow>` for each element in array. `prefixIterator` it's only name of iterator which is used in iteration through array of elements. 

`<TableBody>{array.map((iterator, index) => 
           <TableRow><TableCell>{iterator.id}</TableCell></TableRow>)}
 </TableBody>`

Reply:
@Item21, what is `tableBodyRow?.parent as any)?.parameters[0]` in this particular case?

What value will be for example in `(tableBodyRow?.parent as any)?.parameters[0]?.getText()`?

Reply:
`<TableBody>{array.map((iterator, index) => <TableRow><TableCell>{iterator.id}</TableCell></TableRow>)} </TableBody>` . In this example value of `(tableBodyRow?.parent as any)?.parameters[0]?.getText()` will be `iterator`